glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -494,6 +494,7 @@ open class GleanInternalAPI internal constructor () {
      * This makes all asynchronous work synchronous so we can test the results of the
      * API synchronously.
      */
+    @VisibleForTesting(otherwise = VisibleForTesting.NONE)
```

Comment:
Good catches. There's at least two classes that are shown in our docs but really shouldn't: [GleanInternalApi](https://mozilla.github.io/glean/javadoc/glean/mozilla.telemetry.glean/-glean-internal-a-p-i/index.html) and [PingUploadWorker](https://mozilla.github.io/glean/javadoc/glean/mozilla.telemetry.glean.scheduler/-ping-upload-worker/index.html). The latter can't be made `internal` otherwise the WorkManager would fail, but I think we're fine with making the first one internal. Any chance you could do it in this PR? Bonus point if you find a way to remove the `PingUploadWorker` from kdocs..

Reply:
Nope, won't work:

```
e: /Users/jrediger/mozilla/src/glean.rs/glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt: (576, 16): 'public' subclass exposes its 'internal' supertype GleanInternalAPI
```

Reply:
Darn, that explains it.

Reply:
Made `PingUploadWorker` disappear. I can't do the same for `GleanInternalApi` because that would also hide all docs when inspecting the `Glean` object.